githubbot/githubbot/http.go

```diff
@@ -202,15 +202,24 @@ func (h *HTTPSrv) formatMessage(convID chat1.ConvIDStr, event interface{}, repo
 
 	case *github.CheckRunEvent:
 		var author username
-		if len(event.GetCheckRun().PullRequests) == 0 {
+		// filter checkrun prs by base (they must be AGAINST THE SUBSCRIBED REPO)
+		var runPRs []*github.PullRequest
+		// the repo object returned by the GetCheckRun call is very sparse, so we really only can check against the api url
+		repoAPIUrl := fmt.Sprintf("https://api.github.com/repos/%s", repo)
+		for _, pr := range event.GetCheckRun().PullRequests {
+			if pr.GetBase().GetRepo().GetURL() == repoAPIUrl {
+				runPRs = append(runPRs, pr)
```

Comment:
we don't need an array here just a single object right? why not just have `var runPR *github.PullRequest` and break the loop once we find one

Reply:
ah true, good point